client/homescreen/style.scss

```diff
@@ -81,7 +81,7 @@
 	}
 
 	.components-card__body {
-		padding: 0;
+		padding: 0 !important;
```

Comment:
I don't think we should overwrite this with `!important` for any woocommerce homescreen card. What [we have done in the past](https://github.com/woocommerce/woocommerce-admin/pull/7411/files) is remove the use of `CardBody` in these instances.
So in this case you could remove the use of `CardBody` within the StatsOverview panel.

Reply:
Thank you @louwie17 for the suggestion and the reference PR 👍 

Updated in https://github.com/woocommerce/woocommerce-admin/pull/7594/commits/ed51df30fc24b94b5e1bb45b8ab42a28cb5a8370